ilastik/applets/counting/countingSerializer.py

```diff
@@ -212,42 +212,65 @@ def _deserialize(self, group, slot):
 class CountingSerializer(AppletSerializer):
     """Encapsulate the serialization scheme for pixel classification
     workflow parameters and datasets.
-
     """
+
     def __init__(self, operator, projectFileGroupName):
-        self.predictionSlot = SerialPredictionSlot(operator.PredictionProbabilities,
-                                                   operator,
-                                                   name='Predictions',
-                                                   subname='predictions{:04d}',)
-        slots = [SerialListSlot(operator.LabelNames),
-                 SerialListSlot(operator.LabelColors, transform=lambda x: tuple(x.flat)),
-                 SerialListSlot(operator.PmapColors, transform=lambda x: tuple(x.flat)),
-                 SerialBlockSlot(operator.LabelImages,
-                                 operator.LabelInputs,
-                                 operator.NonzeroLabelBlocks,
-                                 name='LabelSets',
-                                 subname='labels{:0}',
-                                 selfdepends=False),
-                 self.predictionSlot, 
-                 SerialBoxSlot(operator.opTrain.BoxConstraintRois,operator.opTrain,
-                              name="Rois",
-                               subname="rois{:04d}"),
-                 SerialBoxSlot(operator.opTrain.BoxConstraintValues,operator.opTrain,
-                              name="Values",
-                               subname="values{:04d}"),
-                 SerialBoxSlot(operator.boxViewer.rois, operator.boxViewer,
-                              name="ViewRois",
-                              subname="viewrois{:04d}"),
-                 SerialCountingSlot(operator.Classifier,
-                                      operator.classifier_cache,
-                                      name="CountingWrappers")
-                ]
-                
-
-
-        super(CountingSerializer, self).__init__(projectFileGroupName,
-                                                            slots=slots)
+        self.predictionSlot = SerialPredictionSlot(
+            operator.PredictionProbabilities,
+            operator,
+            name='Predictions',
+            subname='predictions{:04d}',
+        )
```

Comment:
Why are you changing this? this is even pep8 correct, or am I missing smth?

Reply:
This is basically for the consistency with the `slots`, and `slots` has multiple constructor calls with long parameter lists, so I thought that hanging indents would be more maintainable.